packages/app/src/runner/selector-playground/Highlight.ce.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <div
+    class="highlight"
+    :style="highlightStyle"
+  />
+  <div
+    class="tooltip"
+    :style="tooltipStyle"
+  >
+    <span>{{ selector }}</span>
+    <div
+      class="arrow"
+      :style="arrowStyle"
+    />
+  </div>
+</template>
+
+<script lang="ts" setup>
+import type { StyleValue, CSSProperties } from 'vue'
+
+const props = defineProps <{
+  selector: string
+  style: StyleValue
+}>()
+
+const highlightStyle = props.style as CSSProperties || {}
+const highlightTop = parseFloat(highlightStyle.top as string)
+const highlightLeft = parseFloat(highlightStyle.left as string)
+const highlightHeight = parseFloat(highlightStyle.height as string)
+const placeOnBottom = highlightTop < 35
+
+const tooltipStyle =
+  placeOnBottom
+    ? {
+      top: `${highlightTop + highlightHeight + 10}px`,
+      left: `${highlightLeft}px`,
+    }
+    : {
+      top: `${highlightTop - 33}px`,
+      left: `${highlightLeft}px`,
+    }
+
+const arrowStyle =
+  placeOnBottom
+    ? {
+      left: `8px`,
+      top: `-6px`,
+      transform: 'rotate(0deg)',
+    }
+    : {
+      left: `8px`,
+      top: `24px`,
+      transform: 'rotate(180deg)',
+    }
+</script>
+
+<style>
```

Comment:
I don't think this is actually needed here - it looks like you grab it via `Highlight.styles` elsewhere. I tried deleting this entire block and just doing `css: `.......` [here](https://github.com/cypress-io/cypress/pull/23973/files#diff-533ffcf1ce12c3f364ebb004a571fce7c2e34670aa16e6b7ac36a1a5b2cd45b9R764) and it still works - thoughts? How about move this out of there into that file (just injecting a string).

Reply:
I placed it here because we usually write related code in one SFC file. 

They're the styles for the highlight and its related elements, I think ce.vue files are better place for them.